ehsm_kms_service/test/test_kms_with_rest.py

```diff
@@ -177,7 +177,7 @@ def test_Stest_RSA3072_sign_verify(base_url, headers):
 
     # test Sign
     payload.clear()
-    payload["digest"] = str(base64.b64encode("test".encode("utf-8")),'utf-8').upper()
+    payload["digest"] = str(base64.b64encode("test".encode("utf-8")),'utf-8')
```

Comment:
nice fix. 
but still same issue @line 217 229  255  267

Reply:
the sign will fix later, since the sign is aligned with server, the sever is using upper now.

Reply:
Seems: https://github.com/intel/ehsm/blob/main/ehsm_kms_service/function.js#L603